servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@uronce-cc Thanks for the advice, I think it's nice to let `PartialLRUEvictor` extends `LRUEvictor`. But there is still a problem now, I have to override `freeSpace(long, BlockStoreLocation, EvictionPlan)` to support select a `StorageDir` first after "cascading eviction" is merged in `master`. So should I let `PartialLRUEvictor` extends `LRUEvictor` now and adjust later or do you have any other suggestions?